sys/include/net/ndp.h

```diff
@@ -84,28 +100,72 @@ extern "C" {
 #define NDP_OPT_PI_FLAGS_MASK       (0xc0)
 #define NDP_OPT_PI_FLAGS_L          (0x80)  /**< on-link */
 #define NDP_OPT_PI_FLAGS_A          (0x40)  /**< autonomous address configuration */
+/** @} */
+
 /**
- * @}
+ * @{
+ * @name    Prefix information option constants
+ * @see     [RFC 4861, section 4.6.2](https://tools.ietf.org/html/rfc4861#section-4.6.2)
  */
+#define NDP_OPT_PI_VALID_LTIME_INF  (UINT32_MAX)    /**< infinite valid lifetime */
+#define NDP_OPT_PI_PREF_LTIME_INF   (UINT32_MAX)    /**< infinite preferred lifetime */
+/** @} */
 
 /**
  * @{
  * @name    Lengths for fixed length options
- * @brief   Options don't use bytes as their length unit, but 8 bytes.
+ * @note    Options don't use bytes as their length unit, but 8 bytes.
  */
 #define NDP_OPT_PI_LEN              (4U)
 #define NDP_OPT_MTU_LEN             (1U)
+/** @} */
+
 /**
- * @}
+ * @{
+ * @name    Router constants
+ * @see     [RFC 4861, section 10](https://tools.ietf.org/html/rfc4861#section-10)
  */
+#define NDP_MAX_INIT_RA_INTERVAL        (16000U)   /**< MAX_INITIAL_RTR_ADVERT_INTERVAL (in msec) */
+#define NDP_MAX_INIT_RA_NUMOF           (3U)       /**< MAX_INITIAL_RTR_ADVERTISEMENT */
```

Comment:
well the RFC tells me the unit of `MAX_INITIAL_RTR_ADVERTISEMENT` is `transmissions` (: